### PR TITLE
solution

### PR DESCRIPTION
**Pull request by Sergej161995**
[DEMO LINK](https://Sergej161995.github.io/layout_stars/)
 [TEST REPORT LINK](https://Sergej161995.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,26 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (ivanchuk-yurii) [inline src/style.css:3]: apply only for elements you need, not `*` all

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -8,6 +8,57 @@
...
+
+      <div class="stars stars--5">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>  
+
+    </div>
```
[the remark this answers is not in the document]
Comment (marina-tilniak) [reply src/index.html:61]: there shouldn’t be empty line between child and parent element

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  
+.stars__star {
+  background-image: url(./images/star.svg);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
```
Comment (marina-tilniak) [inline src/style.css:18]:
```suggestion
  
```

File: src/index.html [hunk 4]
```diff
@@ -8,6 +8,57 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div>
+
+      <div class="stars stars--0">
```
Comment (marina-tilniak) [inline src/index.html:13]:
```suggestion
    <div>
      <div class="stars stars--0">
```

File: src/style.css (hunk 3 above)
Comment (Sergej161995) [reply src/style.css:18]: изменил

File: src/index.html (hunk 4 above)
Comment (TarasHoliuk) [reply src/index.html:13]: You didn't do suggested changes

File: src/index.html [hunk 5]
```diff
@@ -8,6 +8,57 @@
...
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>  
+
+    </div>
+    </div>
   </body>
-</html>
+</html>
```
[the remark this answers is not in the document]
Comment (TarasHoliuk) [reply src/index.html:63]: You didn't add empty line at the end of file - fix it

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Please, fix previous comments

**Pull request closed without merging**